hash.c

```diff
@@ -6842,6 +6842,138 @@ env_update(VALUE env, VALUE hash)
+ *  #has_value?:: Returns +true+ if a given object is a value in +self+, otherwise +false+.
+ *  #include?:: Returns +true+ if a given object is a key in +self+, otherwise +false+.
+ *  #key?:: Returns +true+ if a given object is a key in +self+, otherwise +false+.
+ *  #length:: Returns the count of entries.
+ *  #member?:: Returns +true+ if a given object is a key in +self+, otherwise +false+.
+ *  #size::  Returns the count of entries.
+ *  #value?:: Returns +true+ if a given object is a value in +self+, otherwise +false+.
+ *
+ *  ==== Methods for Comparing
+ *
+ *  {#<}[#method-i-3C]:: Returns +true+ if +self+ is a proper subset of a given object, +false+ otherwise.
+ *  {#<=}[#method-i-3C-3D]:: Returns +true+ if +self+ is a subset of a given object, +false+ otherwise.
+ *  {#==}[#method-i-3D-3D]:: Returns +true+ if a given object is equal to +self+, +false+ otherwise.
+ *  {#>}[#method-i-3E]:: Returns +true+ if +self+ is a proper superset of a given object, +false+ otherwise
+ *  {#>=}[#method-i-3E-3D]:: Returns +true+ if +self+ is a proper superset of a given object, +false+ otherwise.
+ *
+ *  ==== Methods for Fetching
+ *
+ *  #[]:: Returns the value associated with a given key.
+ *  #assoc:: Returns a 2-element array containing a given key and its value.
+ *  #dig:: Returns the object in nested objects that is specified by a given key and identifiers.
+ *  #fetch:: Returns the value for a given key.
+ *  #fetch_values:: Returns array containing the values associated with given keys.
+ *  #key:: Returns the key for the first-found entry with a given value.
+ *  #keys:: Returns an array containing all keys in +self+.
+ *  #rassoc:: Returns a 2-element array consisting of the key and value
+              of the first-found entry having a given value.
+ *  #slice:: Returns a hash containing the entries for given keys.
+ *  #values:: Returns an array containing all values in +self+/
+ *  #values_at:: Returns an array containing values for given keys.
+ *
+ *  ==== Methods for Assigning
+ *
+ *  #[]=:: Associates a given value with a given key.
+ *  #merge:: Returns the hash formed by merging each given hash into a copy of +self+.
+ *  #merge!:: Merges each given hash into +self+.
+ *  #replace:: Replaces the entire contents of +self+ with the contents of a givan hash.
+ *  #store:: Associates a given value with a given key.
+ *  #update:: Merges each given hash into +self+.
+ *
+ *  ==== Methods for Deleting
+ *
+ *  These methods remove entries from +self+:
+ *
+ *  #clear:: Removes all entries from +self+.
+ *  #compact!:: Removes all +nil+-valued entries from +self+.
+ *  #delete:: Removes the entry for a given key.
+ *  #delete_if:: Removes entries selected by a given block.
+ *  #filter!:: Removes entries selected by a given block.
+ *  #keep_if:: Removes entries selected by a given block.
+ *  #reject!:: Removes entries selected by a given block.
+ *  #select!:: Removes entries selected by a given block.
```

Comment:
Please use different text for methods that aren't aliases.  For `#filter, #keep_if, #select`, I would use "Removes entries not selected by a given block" or "Only keep entries selected by a given block".

Reply:
Done, I think, but please re-review.